config/env.development

```diff
@@ -33,6 +33,9 @@ WEB_URL=http://localhost:8000
 # The API Version, used as a prefix on all routes: /v1
 API_VERSION=v1
 
+# GIT_COMMIT, used to retrieve data on the current commit on the master branch, set to master for local enviroments only
+GIT_COMMIT=master
```

Comment:
Is that the reason why we have a message when run `pnpm services:start` like this 
```
time="2022-04-05T09:23:06-04:00" level=warning msg="The \"GIT_COMMIT\" variable is not set. Defaulting to a blank string."
```